packages/http-client-csharp/generator/TestPlugin/src/OutputTypes/TestPluginOutputLibrary.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Generator.CSharp.ClientModel.Providers;
+using Microsoft.Generator.CSharp.Providers;
+
+namespace Microsoft.Generator.CSharp.TestPlugin
```

Comment:
nit: Can we call this SamplePlugin since Test makes it sound like we are testing something.  Or name it by the scenario Sample.OverridePropertySetter or something like this?